tests/phpunit/integration/Core/Conversion_Tracking/Conversion_Event_Providers/Easy_Digital_DownloadsTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @package   Google\Site_Kit
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Tests\Core\Conversion_Tracking\Conversion_Event_Providers;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Assets\Script;
+use Google\Site_Kit\Core\Conversion_Tracking\Conversion_Event_Providers\Easy_Digital_Downloads;
+use Google\Site_Kit\Tests\TestCase;
+
+class Easy_Digital_DownloadsTest extends TestCase {
+
+	/**
+	 * Easy_Digital_Downloads instance.
+	 *
+	 * @var Easy_Digital_Downloads
+	 */
+	private $contactform;
```

Comment:
This is clearly copy-pasted from another test case but is also repeated several times throughout. How was this missed?